CONTRIBUTING.md

```diff
@@ -53,7 +53,7 @@ Notice the `Author` and `Signed-off-by` lines match. If they don't your PR will
 
 1. Changes are automatically linted and tested using the [`ct` tool](https://github.com/helm/chart-testing) as a [GitHub action](https://github.com/helm/chart-testing-action). Those tests are based on `helm install`, `helm lint` and `helm test` commands and provide quick feedback about the changes in the PR. For those tests, the chart is installed on top of [kind](https://github.com/kubernetes-sigs/kind) and this step is not blocking (as opposed to 3rd step).
 2. Changes are manually reviewed by Bitnami team members.
-3. Once the changes are accepted, the PR is tested (if needed) into the Bitnami CI pipeline, the chart is installed and tested (verification and functional tests) on top of different k8s platforms.
+3. Once the changes are accepted, the PR is tested as a part of the Bitnami CI pipeline. The chart is installed and tested (verification and functional tests) on top of different k8s platforms. [Static analysis](https://github.com/bitnami/charts/blob/master/TESTING.md#Static-analysis). is performed as well, including security vulnerability and lint checks. 
```

Comment:
there is a dot after the link.

Some improvement:

3. Once the changes are accepted, the PR is verified with a [Static analysis] that includes the lint and the vulnerability checks. If that passes, the Bitnami team will review the changes and trigger the verification and functional tests.

Reply:
Suggestion applied!